### ISSUE TITLE
job-originating-user-name can be either nameWithLanguage or nameWithoutLanguage

### ISSUE THREAD

**Issue by 14mRh4X0r** [Zelenov/SharpIpp#1]
First, let me thank you for creating this library just when I needed it. Now for the issue:

According to [section 5.3.6 of RFC 8011](https://tools.ietf.org/html/rfc8011#section-5.3.6), `job-originating-user-name` is of type `name(MAX)`, which, according to [section 5.1.3](https://tools.ietf.org/html/rfc8011#section-5.1.3), can be either `nameWithLanguage` or `nameWithoutLanguage`. Currently, SharpIpp defines it as `StringWithLanguage?`, which causes trouble when communicating with [CUPS](https://www.cups.org), which returns it as `nameWithoutLanguage`.

**Comment by Zelenov** [comment]
Great remark, will try to intruduce multiple ways of presenting ` job-originating-user-name`

**Comment by Zelenov** [comment]
Fixed in version [0.0.9](https://www.nuget.org/packages/SharpIpp/0.0.9).
`JobOriginatingUserName ` is now split in two optional parts
```
public string? JobOriginatingUserName { get; set; }
/// <summary>
/// if <see cref="JobOriginatingUserName"/> had a language, this property stores it
/// </summary>
public string? JobOriginatingUserNameLanguage { get; set; }
```
